site/content/docs/main/volume-group-snapshots.md

```
@@ -0,0 +1,379 @@
+# Individual Snapshots (standard CSI behavior)
+velero backup create app-backup --include-namespaces my-app
+```
+
+### The Backup Process
+
+The VGS backup workflow is triggered by a simple label on your PVCs.
+
+1.  **Grouping PVCs:** When a backup is initiated, Velero's `PVCAction` plugin scans for PVCs with the VGS label (the default is `velero.io/volume-group`). All PVCs within the same namespace that share the same label value are collected into a single `ItemBlock`. This ensures they are processed as a single, atomic unit.
+
+2.  **Orchestrating the Snapshot:** The CSI plugin takes over to manage the snapshot creation:
+    *   **Driver Verification:** It first confirms that all PVCs in the group are managed by the same CSI driver.
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)